src/NServiceBus.Hosting.Windows/StartableAndStoppable/StartableAndStoppableTask.cs

```diff
@@ -0,0 +1,26 @@
+namespace NServiceBus
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Features;
+
+    class StartableAndStoppableTask : FeatureStartupTask
```

Comment:
Nest the feature startup task class inside the feature class, more easy to see how it fits together then